api/v1/controllers/bots.js

```diff
@@ -197,7 +197,7 @@ module.exports = {
     u.findByKey(helper, req.swagger.params)
     .then((o) => {
       o.set('lastHeartbeat', timestamp);
-      res.status(httpStatus.OK);
+      res.status(httpStatus.OK).json();
```

Comment:
Where's the test which shows that this change makes it work?

Reply:
@iamigo test added